readme.md

```diff
@@ -230,6 +256,5 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
+// Your answer... really lost when it comes to the MEAN stack. As stated above, I'll be spending time with it this week.
 ```
```

Comment:
Yup, this can be tough. I found it helpful to recreate the table we had in one of the lesson plans and find equivalencies between Active Record and Mongoose. From there, it's just a matter of hammering out the syntax. Let me know if you want any help with this after reviewing.